### PR TITLE
Remove tsup, fix type output with tsc-alias

### LINKED ISSUE #135: `startServer` in node adapter has untyped result
@codetheweb

image

### PR DESCRIPTION
**Pull request by codetheweb**
Tested in fake-template.

### CODE REVIEW

File: src/cli/cli.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env node
 import { runExit } from "clipanion"
-import { version } from "../../package.json"
```
Comment (codetheweb) [inline src/cli/cli.ts:3]: couldn't get .json imports to work easily

File: src/edge/transform-to-node.ts
Comment (codetheweb) [inline src/edge/transform-to-node.ts:1]: had to move this so that tsc-alias didn't confuse importing `node_modules/edge-runtime` with `src/edge-runtime`

**Review by mxsdev — APPROVED** [review]
👍 

did you confirm this fixes type issues?

**Comment by codetheweb** [comment]
yep

**Pull request merged**